free-courses-en.md

```diff
@@ -97,6 +98,9 @@
 * [Bento Shell Track](https://bento.io/topic/shell) (Bento)
 * [Shell Scripting Tutorial](https://www.youtube.com/playlist?list=PL7B7FA4E693D8E790)
 
+### BlockChain
+* [Bitcoin and Cryptocurrency Technologies](https://www.lopp.net/pdf/princeton_bitcoin_book.pdf) - Bitcoin and Cryptocurrency Technologies
```

Comment:
duplicate of book in ###language_agnostic####misc; why do you think "lopp.net" is an official source?